aws-logs-destination/src/test/java/software/amazon/logs/destination/UpdateHandlerTest.java

```diff
@@ -182,4 +182,45 @@ public void handleRequest_Should_ReturnSuccess_When_DestinationIsFound() {
         assertThat(response.getErrorCode()).isNull();
     }
 
+    // tests for optional parameter destination policy not provided tests
+    @Test
+    public void handleRequest_Should_ReturnSuccess_When_DestinationIsFound_and_DestinationPolicyNotProvided() {
+        final Destination destination_wihtout_policy = getTestDestination(false);
```

Comment:
nit: typo and should be camelCase

Reply:
not sure why VS Code didn't point that one out, thanks!

Reply:
oh I see, its a different file, I defined it with the typo